Components/Windows/configure-wsus-server/configure-wsus-server.yaml

```diff
@@ -0,0 +1,114 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
```

Comment:
This component is not owned by Amazon, but rather the community, so the first line of the header should not be included. The second line is optional.